packages/blocks/src/api/test/factory.js

```diff
@@ -2276,4 +2277,33 @@ describe( 'block factory', () => {
 			expect( isContainerGroupBlock( 'core/group' ) ).toBe( false );
 		} );
 	} );
+
+	describe( 'getBlockFromExample', () => {
```

Comment:
This test is similar to this test:

https://github.com/WordPress/gutenberg/blob/b6b4cf6bff9fd4052fa62a42c6027d2c821b630a/packages/blocks/src/api/test/templates.js#L209-L219